static/panes/compiler.ts

```diff
@@ -2501,19 +2513,26 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
                     (button.prop('disabled') ? ' [LOCKED]' : '')
             );
         };
-        const isIntelFilterDisabled = !this.compiler.supportsIntel && !filters.binary;
+        const isIntelFilterDisabled = !this.compiler.supportsIntel && !filters.binary && !filters.binaryobject;
         this.filterIntelButton.prop('disabled', isIntelFilterDisabled);
         formatFilterTitle(this.filterIntelButton, this.filterIntelTitle);
-        // Disable binary support on compilers that don't work with it.
-        this.filterBinaryButton.prop('disabled', !this.compiler.supportsBinary);
+
+        // Disable binaryobject support on compilers that don't work with it or if binary is selected
```

Comment:
I'll have to try out the UI locally but does this mean once you pick binary you can't pick binaryObject until you unselect binary?

Reply:
You're probably right. I'll need to read how this can be done, CSS/HTML/TS/PUG/whatever is not really in my comfort zone :)

Reply:
checking/de-checking with code is not trivial unfortunately, I've looked into it a couple of times and spent way too much time on it without achieving anything, so would advice to stay clear of that

Reply:
Ok, so I think I'll skip this suggestion, sorry @Alcaro . Unless you have some hint on how to achieve this of course

Reply:
I know how to programmatically manipulate checkboxes in plain old HTML (that's the .checked attribute), but I have no idea about PUG or Typescript or any other templating engine. Those checkboxes aren't even checkboxes, they're <button>.

Buttons can be manipulated with .click() if you can access the HTMLButtonElement object, but I don't know if your templating engine allows that. And click event handlers triggering other click event handlers counts as recursion, which isn't necessarily a good idea.

Either way, if this is difficult, let's not let it block the rest of the feature. It would be an improvement, but we can spin it off as a separate issue.

Reply:
That's really too far from what I can expect to do now, so I'll trust @partouf and leave it as a possible enhancement for later®